WineskinLauncher/WineskinLauncherAppDelegate.m

```diff
@@ -2043,14 +1887,24 @@ - (void)startWine:(WineStart *)wineStartInfo
                     [self systemCommand:[NSString stringWithFormat:@"chmod -h 777 \"%@\"",usersUserFolderPath]];
                 }
                 
+                // TODO: copy remakedefaults.reg to /tmp edit as needed instead of using a second copy
```

Comment:
I was quickly testing just using another remakedefaults.reg depending on if XQuartz was installed, as you can see with the TODO, should instead copy to /tmp and edit the line as needed depending on if XQuartz is installed or not.